articles/multifactor-authentication/api/email.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Associate an Email Authenticator
+description: Configure your application so users can self-associate email authenticators.
+topics:
+  - mfa
+  - mfa-api
+  - mfa-authenticators
+  - email
+contentType:
+  - how-to
+useCase:
+  - customize-mfa
+---
+
+# Associate an Email Authenticator
+
+In this tutorial, you'll learn how to configure your application so users can use email authenticators.
+
```

Comment:
Need to add an info or warning box here describing that email MFA only works with this API and is not supported as part of the hosted mfa page. This means that users enrolled for email can only use it as an authenticator with apps that consume the API and not with those that redirect the user to the hosted page.